index.html

```diff
@@ -126,6 +126,16 @@ <h1>Supported:</h1>
                                                                           <td><a href="https://github.com/mit-cml/appinventor-extensions/tree/extension/personal-image-classifier">Via GitHub</a></td>
                                                                         </tr>
 
+									<tr>
+                                                                          <td>PosenetExtension</td>
+                                                                          <td>
+                                                                            Estimate pose with this extension.
+                                                                          </td>
+                                                                          <td>MIT App Inventor</td>
+                                                                          <td>20200226</td>
+                                                                          <td><a href="data/extensions/edu.mit.appinventor.ai.posenet.aix">Posenet.aix</a></td>
+                                                                          <td><a href="https://github.com/ewpatton/appinventor-sources/tree/extension/posenet">Via GitHub</a></td>
```

Comment:
This branch should be in the extensions repo now, so you should reference that version instead of the one in my personal repo.

Reply:
Updated.